central/policy/datastore/datastore_impl_test.go

```diff
@@ -44,9 +44,7 @@ type PolicyDatastoreTestSuite struct {
 }
 
 func (s *PolicyDatastoreTestSuite) SetupTest() {
-	if env.PostgresDatastoreEnabled.BooleanSetting() {
-		s.T().Skip("Tests are not applicable when ROX_POSTGRES_DATASTORE is set to true")
-	}
+	pgtest.SkipIfPostgresEnabled(s.T())
```

Comment:
Add TODO if this test will be updated or removed separately.  I'm guessing ROX-17689

Reply:
yeah this will need to be addressed with policies